notNeededPackages.json

```
@@ -380,6 +380,10 @@
             "libraryName": "@braintree/sanitize-url",
             "asOfVersion": "4.1.0"
         },
+        "bree": {
+            "libraryName": "bree",
+            "asOfVersion": "7.1"
```

```suggestion
            "asOfVersion": "7.1.0"
```

as per https://github.com/breejs/bree/releases/tag/v7.1.0

---

(this will make the CI happy)